src/commons/api/roas.rs

```diff
@@ -129,29 +129,35 @@ impl RoaAggregateKeyFmtError {
     }
 }
 
-//------------ RoaDefinition -----------------------------------------------
+//------------ RoaPayload --------------------------------------------------
 
-/// This type defines the definition of a Route Origin Authorization (ROA), i.e.
-/// the originating asn, IPv4 or IPv6 prefix, and optionally a max length.
+/// This type defines the definition of a Route Origin Authorization (ROA)
+/// payload: ASN, Prefix and optional Max Length
+///
+/// Note that a ROA object may contain multiple payloads (RoaDefinitions)
```

Comment:
This is confusing as the comment has been updated but refers to the no-longer-existing RoaDefinition type.

Reply:
fixed